recipes/openmoltools/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "openmoltools" %}
+{% set version = "0.8.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/choderalab/openmoltools/archive/{{ version }}.tar.gz
+  sha256: dae37dc1136cbc6175b1935e2b4352cc7cc9c9e4d441968178c9e90ec674a1df
+  patches:
+  - setup.patch
```

Comment:
What is this patch doing?

Reply:
I removed `import numpy` and other unnecessary imports, but looks like autoformatting got in the way and the diff is noisier than needed. I will simplify it.

Reply:
See now :)